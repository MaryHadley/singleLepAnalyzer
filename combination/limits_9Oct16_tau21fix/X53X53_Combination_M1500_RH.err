/var/lib/condor/execute/dir_15261/condor_exec.exe: line 6: cd: limits_9Oct16_tau21fix/: No such file or directory
/cvmfs/cms.cern.ch/slc6_amd64_gcc491/external/py2-matplotlib/1.2.1-cms2/lib/python2.7/site-packages/matplotlib/axes.py:2542: UserWarning: Attempting to set identical left==right results
in singular transformations; automatically expanding.
left=0.0, right=0
  + 'left=%s, right=%s') % (left, right))
