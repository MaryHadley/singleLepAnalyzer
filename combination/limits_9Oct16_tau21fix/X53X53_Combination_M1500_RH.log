000 (1295363.000.000) 10/09 22:41:04 Job submitted from host: <192.168.63.249:41533>
...
001 (1295363.000.000) 10/09 22:41:05 Job executing on host: <192.168.63.239:46789>
...
006 (1295363.000.000) 10/09 22:41:14 Image size of job updated: 1
	3  -  MemoryUsage of job (MB)
	2248  -  ResidentSetSize of job (KB)
...
006 (1295363.000.000) 10/09 22:46:15 Image size of job updated: 282984
	141  -  MemoryUsage of job (MB)
	144356  -  ResidentSetSize of job (KB)
...
006 (1295363.000.000) 10/10 00:01:22 Image size of job updated: 282984
	142  -  MemoryUsage of job (MB)
	144440  -  ResidentSetSize of job (KB)
...
006 (1295363.000.000) 10/10 17:03:08 Image size of job updated: 297580
	158  -  MemoryUsage of job (MB)
	161468  -  ResidentSetSize of job (KB)
...
005 (1295363.000.000) 10/10 17:07:49 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 18:25:18, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 18:25:18, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1596  -  Run Bytes Sent By Job
	307  -  Run Bytes Received By Job
	1596  -  Total Bytes Sent By Job
	307  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :    31872        1   3824810
	   Memory (MB)          :      158     3072      3072
...
