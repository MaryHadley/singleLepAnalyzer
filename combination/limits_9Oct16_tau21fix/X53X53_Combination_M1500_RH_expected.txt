# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0    0.06919   0.034972    0.14324  0.047907    0.10137
